### ISSUE TITLE
Export the same CVE ID and pakcage issue

### ISSUE THREAD

**Issue by YangJiao0817** [goharbor/harbor#17188]
**Expected behavior and actual behavior:**
Expected:Export CVE data is the same as the UI page
Actual:Export CVE data is different from UI page.
Two records with the same CVE id and the same package, but with different versions, will only show one record in the csv file.

**Steps to reproduce the problem:**
1. Create a project as test02
2. Push image(goharbor/notary-server-photon:v2.5.3) to test02
2. Export CVE data
<img width="595" alt="图片" src="https://user-images.githubusercontent.com/72076317/179455664-9f7cbab9-5fd2-45bf-a6be-d64e1c314126.png">
3. Compare CVE data
<img width="1152" alt="图片" src="https://user-images.githubusercontent.com/72076317/179458197-5b69aebe-001a-4807-8bc1-4b150d3d3f26.png">
<img width="1141" alt="csv_file" src="https://user-images.githubusercontent.com/72076317/179458024-46cb628c-f2c5-4263-9d51-ba769433ac77.png">

But even if the data in the csv_file is complete, they cannot be distinguished in the csv_file, because they differ only in the version, and we should add the version columns.
Open this issue(https://github.com/goharbor/harbor/issues/17187) to discuss add version columns.

**Versions:**
- harbor version: v2.6.0-f3edb03b

**Comment by prahaladdarkin** [comment]
@YangJiao0817  @AllForNothing  @wy65701436  - this will get resolved if we add the package version and the fix version columns to the CSV export.

**Comment by AllForNothing** [comment]
@prahaladdarkin And also notice that the `CVSS3` score column is empty, can you also add this to the CSV file

**Comment by prahaladdarkin** [comment]
@AllForNothing  @wy65701436  - One point about the CVSS3 and CVSS2 columns - the data for the CVSS values comes from the column `vendor_attributes` which contains the vendor specific CVSS data. Trivy would publish the vulnerability data from it's sources NVD and others within the `vendor_attributes` column. There could be other vulnerability scanners that would not necessarily publish any of the data within the vendor attributes table.
At this point, one question then arises - how to reliably look for CVSS3 or CVSS2 information? I recollect during the discussion of schema normalization, that this approach was adopted because some other partners were using the CVSS field in a specific way and we did not want to break backward compatibility. The vendor attributes column was the best fit to put in additional vendor specific data (this also allows vendors to put in multiple sources of CVSS data). 
IMHO, since the UI displays this data by parsing the contents of the vendor attributes table, we can do the same for the CSV export. We need to put this data into a column labeled `Additional Data`. For this specific release, we can remove specific to CVSS scores from the report. This way the CSV report would be consistent with the UI. let me know your views. Please refer below for a sample excerpt of the `vendor_attributes` column
```{
  "CVSS": {
    "nvd": {
      "V2Score": 6.8,
      "V2Vector": "AV:N/AC:M/Au:N/C:P/I:P/A:P",
      "V3Score": 7.8,
      "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H"
    },
    "redhat": {
      "V3Score": 5.8,
      "V3Vector": "CVSS:3.1/AV:L/AC:H/PR:L/UI:N/S:U/C:L/I:L/A:H"
    }
  }
}```

**Comment by AllForNothing** [comment]
@prahaladdarkin I agree being consistent with the UI

**Comment by prahaladdarkin** [comment]
Based on discussion with @wy65701436  @AllForNothing  @chlins  the following changes are already available as a part of PR https://github.com/goharbor/harbor/pull/17232
* Presence of package version and fixed version columns
For the CVSS3 scores, owing to the vendor specific mechanism of storing CVSS scores from various sources, @wy65701436 will discuss this with the scanner team for further clarity.

**Comment by zyyw** [comment]
Hi @prahaladdarkin , 
Instead of having columns `CVSS V3 Score`, `CVSS V2 Score`, `CVSS V3 Vector` and `CVSS V2 Vector` in the cve data export files, we think it might be better to just dump the data from `vendor_attributes` into a column name `Additional Data`. Could you please help us to have this change implemented in the coming Harbor 2.6.0 release? Thank you.
